app/models/post.rb

```diff
@@ -0,0 +1,23 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
```

Comment:
- Up :arrow_up: you said a user has many posts and didn't specified explicitly author as the user. I think this line of code is only correct if you made the changes above :+1: